docker-compose.yml

```diff
@@ -11,17 +11,62 @@ services:
             - pg_data:/var/lib/postgresql/data
         ports:
             - "${PG_PORT}:5432"
-        
+    db_exporter:
+        image: quay.io/prometheuscommunity/postgres-exporter
+        environment:
+            - DATA_SOURCE_NAME=postgresql://isb_writer:isamplesinabox@db:5432/postgres?sslmode=disable
+        expose:
```

Comment:
The `expose` syntax was one I was not previously familiar with.  It opens up the port on the Docker network but doesn't bind to the host machine.